FF1Lib/Scale.cs

```diff
@@ -440,30 +440,15 @@ public void SetProgressiveScaleMode(Flags flags)
 			PutInBank(0x1B, 0x806D, Blob.FromHex("20CBCFEAEAEAEAEA"));
 		}
 
-		private void ScaleEncounterRate(double overworldMultiplier, double dungeonMultiplier)
+		
+		private void ScaleEncounterRate(double LandEncounterRate, double SeaEncounterRate, double DungeonEncounterRate)
 		{
-			if (overworldMultiplier == 0)
-			{
-				PutInBank(0x1F, 0xC50E, Blob.FromHex("EAEA"));

```

Comment:
You want to keep the code patches -- I believe just setting a value of 0 means you get 1/256 encounters.